docs/ecosystem/changelog.md

```diff
@@ -0,0 +1,22 @@
+---
+id: changelog
+title: Changelogs
+---
+
+> Running security-critical open source technology in a self-service format requires vigilance. We make your life easier by
+> announcing important software updates via the [Ory Security Newsletter](http://eepurl.com/di390P). Never miss an update and
+> **[sign up now to important release updates!](http://eepurl.com/di390P)**
+
+Ory is actively being developed. If you are interested in future development, take a look at the [Roadmap](./roadmap.md). Below
+you find changelogs for Ory services:
```

Comment:
```suggestion
Ory is being actively developed. If you want to know more about the planned features and timelines, read the [Roadmap](./roadmap.md).

Visit these links to get changelogs of individual Ory projects:
```

Reply:
> Ory is actively being developed.

I have a problem with that. Is ORY something we develop? Or are Ory PROJECTS or PRODUCTS something we develop? 🤔 Sounds unimportant but I think we should settle on one way of talking about these things and then stick to it.

Reply:
Ah you are right. 
But I think in this case "Ory" means the entirety of the "ecosystem" - all open source repos and other products.